doc/fidget.md

```diff
@@ -82,6 +82,11 @@ The following table shows the default options for this plugin:
     fidget_decay = 2000,      -- how long to keep around empty fidget, in ms
     task_decay = 1000,        -- how long to keep around completed task, in ms
   },
+  window = {
+    relative = "editor",      -- where to anchor the window, either `"win"` or `"editor"`
```

Comment:
This table shows the default options, so it should be `"win"` here (and `blend = 100` and `zindex = nil` below).